object-store/1.2/modules/ROOT/pages/object-store-to-define-a-new-os.adoc

```diff
@@ -1,46 +1,61 @@
 = Define Object Stores Examples - Mule 4
 :page-aliases: connectors::object-store/object-store-to-define-a-new-os.adoc
 
-You can define an Object Store globally in the app so it can be referenced by name and shared between multiple components, or you can create an Object Store that only for a particular component.
+You can define an Object Store globally in the app so it can be referenced by name and shared between multiple components, or you can create an Object Store for one particular component.
 
-All Object Store operations can reference a global Object Store. For example, suppose you want to have an audit flow that verifies whether certain users are currently authorized. You might do something like this to reference the Object Store that you have defined:
+All Object Store operations can reference a global Object Store. For example, for an audit flow that verifies whether certain users are currently authorized, you can reference the Object Store that you defined:
```

Comment:
"that you defined" - where was this object store defined? Maybe you can modify the lead-in a bit to say "...you can define and reference a global object store named `tokenStore`.